devito/ir/iet/nodes.py

```diff
@@ -521,6 +534,31 @@ def write(self):
         return []
 
 
+class While(Node):
+
+    """
+    Implement a while-loop.
```

Comment:
why `-`?

Reply:
because I used 'for-loop' in the `Iteration` docstring. Probably because it just went out that way. I can kill the hyphen

Reply:
Just me being picky, people use way too many hyphens for my taste :D